samples/net/sockets/echo_client/boards/intel_socfpga_agilex5_socdk.conf

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022-2023, Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MDIO=y
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_PKT_RX_COUNT=64
+CONFIG_NET_PKT_TX_COUNT=64
+CONFIG_NET_BUF_RX_COUNT=1024
+CONFIG_NET_BUF_TX_COUNT=512
+
+CONFIG_ISR_STACK_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_IDLE_STACK_SIZE=2048
+CONFIG_SHELL_STACK_SIZE=10240
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+CONFIG_NET_MGMT_EVENT_STACK_SIZE=32768
+CONFIG_NET_TCP_WORKQ_STACK_SIZE=4096
+CONFIG_NET_RX_STACK_SIZE=32768
```

Comment:
The device seems to have lot of ram, but I am still wondering why do we have these numbers. I mean for example the net_mgmt stack size 32kb, where did this number came from. Have you measured how much stack is being used by the net_mgmt thread?

Reply:
Hello @jukkar, I concur that these RAM size figures are indeed atypical. The rationale for overriding the default configured values is due to specific requests from our customer support team for all the applications supported on our platform. The requirement is that the application should be built with NO_OPTIMIZATION enabled. The default stack sizes were initially estimated for the SIZE_OPTIMIZATION option. However, when we tested the applications with the default stack sizes, we encountered stack overflow issues. Subsequently, we attempted to increase the stack size until the applications functioned properly.

Reply:
We have tested client-server applications with default stack size values and with default optimization values, and they are working fine.

Reply:
@jukkar After removing all the additional stack configurations and compiling with the default optimization level I am facing an issue with the echo_client application. I have added consoles screen shots and wireshark packet dump here
![image](https://github.com/zephyrproject-rtos/zephyr/assets/94606265/4d30fb83-0895-4d90-946f-d97fdfa8c720)
![image](https://github.com/zephyrproject-rtos/zephyr/assets/94606265/5e674760-a5ed-4713-b1ff-f6a447e00549)

Reply:
But it was working if I keep CONFIG_NET_MGMT_EVENT_STACK_SIZE=32768

Reply:
I do not mind using the stack values for this board, just wondering why they were so large.